README.md

```diff
@@ -76,7 +76,7 @@ This will compile your project and store the build artifacts in the `dist/` dire
 
 ## Running unit tests
 
-To execute unit tests with the [Karma](https://karma-runner.github.io) test runner, use the following command:
+To execute unit tests with the [Jest](https://jestjs.io/) testing framework, use the following command:
 
 ```bash
 ng test
```

Comment:
This is incorrect no?

Reply:
Good point, yes, this is a relic from the auto generated documentation. The new section on unit testing is further below. I will add a documentation clean-up task to the next sprint to remove all these auto-generated parts and proof-read the whole documentation.